### DISCUSSION TITLE
Proxy error: tls

### DISCUSSION THREAD

**Discussion by CLtheorem** [dstotijn/hetty#73] [General] [answered]
I just download the latest release file and run it directly. I got the following error when I go to http://127.0.0.1:8080 (HTTP ERROR 502). It happened in  both mac and linux(kali).
<img width="688" alt="截屏2021-01-12 上午10 50 41" src="https://user-images.githubusercontent.com/34810210/104263200-0fc7af00-54c4-11eb-9d43-25dc228120e9.png">
is this a bug or just something wrong with my operation?

**Comment by lalyos** [comment, marked as answer]
use http://localhost:8080 instead of 127.0.0.1
